src/main/java/com/saralein/server/response/Header.java

```diff
@@ -1,30 +1,49 @@
 package com.saralein.server.response;
 
 import static com.saralein.server.Constants.CRLF;
+import static com.saralein.server.Constants.STATUS_CODES;
+import java.util.HashMap;
 
-class Header {
-    private final String status;
-    private final String contentType;
+public class Header {
+    private final String version = "HTTP/1.1";
+    private HashMap<String, String> header;
 
-    Header(String status, String contentType) {
-        this.status = status;
-        this.contentType = contentType;
+    public Header() {
+        this.header = new HashMap<>();
     }
 
-    private String createStatusLine(String status) {
-        return  "HTTP/1.1 " + status + CRLF;
+    public void addStatus(int code) {
+        header.put("Status", createStatusLine(code));
     }
 
-    private String createContentTypeHeader(String contentType) {
-        return "Content-Type: " + contentType + CRLF;
+    public void addHeader(String title, String content) {
+        header.put(title, content);
     }
 
-    String createContents() {
+    public byte[] convertToBytes() {
+        return createContents().getBytes();
+    }
+
+    private String createStatusLine(int code) {
+        return  version + " " + STATUS_CODES.get(code);
+    }
+
+    private String createContents() {
```

Comment:
There is a lot going on here. Maybe you could extract some methods with intention-revealing names?